bin/tx-prover/src/lib.rs

```diff
@@ -4,6 +4,8 @@ use alloc::string::String;
 
 pub mod generated;
 
+#[cfg(not(target_arch = "wasm32"))]
+pub mod error;
```

Comment:
Why don't we want to export the errors for WASM?

Reply:
This is a mistake. The error is only for std, but it should not be here in the `lib.rs`. I'm moving it to `main.rs`.

Reply:
That's because it is only used in the CLI and Proxy, and does not support WASM.